pkg/preparation/sqlrepo/uploads.go

```diff
@@ -166,14 +161,7 @@ func (r *repo) CIDForFSEntry(ctx context.Context, fsEntryID id.FSEntryID) (cid.C
 		return cid.Undef, err
 	}
 	if ds.State() != dagmodel.DAGScanStateCompleted {
-		switch ds.State() {
```

Comment:
I can't remember why I thought these should be nil error for Failed & Canceled, but should they be nil?